packages/opentelemetry-basic-tracer/src/Span.ts

```diff
@@ -116,13 +120,29 @@ export class Span implements types.Span {
     if (this._isSpanEnded()) return;
     this._ended = true;
     this._endTime = endTime || performance.now();
-    // @todo: record or export the span
+    this.spanProcessor.onEnd(this);
   }
 
   isRecordingEvents(): boolean {
     return true;
   }
 
+  toReadableSpan(): ReadableSpan {
+    return {
+      name: this._name,
+      traceId: this._spanContext.traceId,
+      spanId: this._spanContext.spanId,
+      parentSpanId: this._parentId,
+      kind: this._kind,
+      startTime: this._startTime,
+      endTime: this._endTime,
+      status: this._status,
+      attributes: this._attributes,
+      links: this._links,
+      events: this._events,
+    };
+  }
+
   toString() {
     const json = JSON.stringify({
```

Comment:
Can we make `toString` here utilize `toReadableSpan` above instead of doing the same thing separately?